view/frontend/web/js/view/payment/method-renderer/omise-offsite-alipay-method.js

```diff
@@ -0,0 +1,122 @@
+define(
+    [
+        'jquery',
+        'ko',
+        'mage/storage',
+        'Magento_Checkout/js/view/payment/default',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/action/redirect-on-success',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/url-builder',
+        'Magento_Checkout/js/model/error-processor'
+    ],
+    function (
+        $,
+        ko,
+        storage,
+        Component,
+        fullScreenLoader,
+        redirectOnSuccessAction,
+        quote,
+        urlBuilder,
+        errorProcessor
+    ) {
+        'use strict';
+
+        return Component.extend({
+            defaults: {
+                template: 'Omise_Payment/payment/offsite-alipay-form'
+            },
+
+            isPlaceOrderActionAllowed: ko.observable(quote.billingAddress() != null),
+
+            /**
+             * Get payment method code
+             *
+             * @return {string}
+             */
+            getCode: function() {
+                return 'omise_offsite_alipay';
+            },
+
+            /**
+             * Initiate observable fields
+             *
+             * @return this
+             */
+            initObservable: function() {
+                this._super()
+                    .observe([
+                        'omiseOffsite'
```

Comment:
@jacekstan I forgot how this one works, but do we still need to observe this one even we don't have any special field at the Alipay payment form?